UserApi/UserApi/Helper/UserProfileHelper.cs

```diff
@@ -25,23 +25,31 @@ public class UserProfileHelper
             {"Civil Money Claims", AdGroup.MoneyClaims}
         };
 
+        private static readonly Dictionary<AdGroup, string> CaseTypeMappings = new Dictionary<AdGroup, string>
+        {
+            { AdGroup.MoneyClaims, "Civil Money Claims" },
+            { AdGroup.FinancialRemedy, "Financial Remedy" }
+        };
+
         public UserProfileHelper(IUserAccountService userAccountService)
         {
             _userAccountService = userAccountService;
         }
 
-        public async Task<UserProfile> GetUserProfile(string filter)
+        public async Task<UserProfile> GetUserProfileAsync(string filter)
         {
             var user = await _userAccountService.GetUserByFilter(filter);
 
             if (user == null)
+            {
                 return null;
+            }
 
             var userGroupDetails = await _userAccountService.GetGroupsForUser(user.Id);
```

Comment:
async name?

Reply:
Indeed, this wasn't done. It's nothing to do with my PR though so I'd rather not blend it in here. I'll raise a separate PR for it if that's good.